fpdf/fpdf.py

```diff
@@ -2446,6 +2530,8 @@ def multi_cell(
             max_line_height (int): optional maximum height of each sub-cell generated
             markdown (bool): enable minimal markdown-like markup to render part
                 of text as bold / italics / underlined. Default to False.
+            print_sh (bool): Treat a soft-hyphen (\\u00ad) as a normal printable
+                                character, instead of a line breaking opportunity. Default value: False
```

Comment:
Just a detail: could you align the start of this costring line to match the "of text as bold" line above please?

Reply:
Vim getting confused about something... will do.